prisma/schema.prisma

```diff
@@ -42,14 +42,16 @@ model sessions {
 }
 
 model users {
-  id             Int       @id @default(autoincrement())
-  name           String?
-  email          String?   @unique
-  email_verified DateTime?
-  image          String?
-  created_at     DateTime  @default(now())
-  updated_at     DateTime  @default(now())
-  players        players[]
+  id                                                       Int                   @id @default(autoincrement())
+  name                                                     String?
+  email                                                    String?               @unique
+  email_verified                                           DateTime?
```

Comment:
Is there a reason why we would want to keep track of when an email has been verified in comparison to just checking if the verification happened/didn't happen?